src/components/VerifyPageContent/helpers.ts

```diff
@@ -4,7 +4,11 @@ import { getChainInfo } from "../../common/utils/chain-utils";
 type LoadCertificate = (certificate: any) => void;
 
 export const getDemoCert = (chainId: ChainId): string => {
-  const networkName = getChainInfo(chainId).networkName;
+  let networkName: string | undefined;
+  try {
+    networkName = getChainInfo(chainId).networkName;
+  } catch (e) {}
+
```

Comment:
should we do something to error?

Reply:
If there is an error, `networkName` will be defaulted to "undefined" for the file name. The only error that can come from here is an invalid network. This only happens if the selected network is an invalid one. Which on the frontend the screen will already switch to the error page and the demo cert won't be available to the user. That's why I thought maybe there isn't much to do if it's caught here.